pom.xml

```diff
@@ -247,7 +247,21 @@
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>okhttp</artifactId>
       <version>3.7.0</version>
-</dependency>
+      <scope>test</scope>
```

Comment:
Why changing this scope?

Reply:
This dependency appeared in https://github.com/SonarSource/sonar-scanner-jenkins/commit/be10664e2519100efe6b356a62e6074b5cb5178a and as you see it is used there only `WaitForQualityGateStepTest`

Reply:
Initially I wanted to improve the scope in a separate PR: https://github.com/SonarSource/sonar-scanner-jenkins/pull/130 but after a discussion https://github.com/SonarSource/sonar-scanner-jenkins/pull/124#issuecomment-545331715 we decided to include it in here.

Reply:
I believe it was actually MEANT to be in test scope, especially if you see that it was added after the comment `<!-- Test deps -->`  https://github.com/SonarSource/sonar-scanner-jenkins/blob/master/pom.xml#L186

Reply:
LGTM @tom-vanbraband-sonarsource do you confirm?

Reply:
Yes, that's my mistake. Thanks for fixing that @Smasherr

Reply:
So, this turns out to be incorrect, as it's used by some other dependency. I'll update this in #132 directly, and merge once the QA passes.